.storybook/utils/componentsResolver.js

```diff
@@ -6,23 +6,94 @@ const componentsDir = path.resolve(__dirname, '../../packages');
 /**
  * Local import
  */
-module.exports = componentResolver = {
+
+
+class WebpackResolveComponentSourcePlugin {
   apply(resolver) {
-    resolver.plugin('module', function (init, callback) {
-      if (init.request.startsWith(componentsPrefix)) {
-        const componentName = init.request.replace(componentsPrefix, '');
-
-        console.log('!componentsPrefix!');
-
-        this.doResolve(
-          'resolve',
-          { ...init, request: `${componentsDir}/${componentName}/src` },
-          `Resolve ${init.request}`,
-          callback,
-        );
-      } else {
-        callback();
+    const target = resolver.ensureHook('resolved');
+
+    resolver
+      .getHook('module')
+      .tapAsync('pluginName', (request, resolveContext, callback) => {
+        const {
+          descriptionFileData,
+          request: requestPath,
+        } = request;
+
+        if (requestPath && requestPath.match(componentsPrefix)) {
+
+          const componentName = requestPath.replace(`./${componentsPrefix}`, '');
+          const path = `${componentsDir}`;
+
+        console.log('requestPath -->', requestPath);
```

Comment:
Do we have any reason for these console.logs, if we don't have to, could you clean please?